File: Basics/Conditional-Statements-Advanced/Exercise/new_house.py
flowers_kind = input()
flowers_count = int(input())
budget = int(input())

price = 0

if flowers_kind == "Roses":
    price = flowers_count * 5
    if flowers_count > 80:
        price -= price * 0.10
elif flowers_kind == "Dahlias":
    price = flowers_count * 3.80
    if flowers_count > 90:
        price -= price * 0.15
elif flowers_kind == "Tulips":
    price = flowers_count * 2.80
    if flowers_count > 80:
        price -= price * 0.15
elif flowers_kind == "Narcissus":
    price = flowers_count * 3
    if flowers_count < 120:
        price += price * 0.15
elif flowers_kind == "Gladiolus":
    price = flowers_count * 2.50
    if flowers_count < 80:
        price += price * 0.20

left_needed_sum = abs(budget - price)
if budget >= price:
    print(f"Hey, you have a great garden with {flowers_count} {flowers_kind} and {left_needed_sum:.2f} leva left.")
else:
    print(f"Not enough money, you need {left_needed_sum:.2f} leva more.")

File: JS-Basics/0-3-Conditional-Steatements-Advanced/Exercise/new-house.js
function isBudgeEnough(input) {
    let flowerType = input[0];
    let flowersCount = Number(input[1]);
    let budget = Number(input[2]);
    let flowersPrices = {
        "Roses": 5,
        "Dahlias": 3.80,
        "Tulips": 2.80,
        "Narcissus": 3,
        "Gladiolus": 2.50
    }
    let price = flowersCount * flowersPrices[flowerType];
    if (flowerType == "Roses" && flowersCount > 80) {
        price -= price * 0.10;
    } else if (flowerType == "Dahlias" && flowersCount > 90) {
        price -= price * 0.15;
    } else if (flowerType == "Tulips" && flowersCount > 80) {
        price -= price * 0.15;
    } else if (flowerType == "Narcissus" && flowersCount < 120) {
        price += price * 0.15;
    } else if (flowerType == "Gladiolus" && flowersCount < 80) {
        price += price * 0.20;
    }

    let diff = Math.abs(price - budget);
    if (price <= budget) {
        console.log(`Hey, you have a great garden with ${flowersCount} ${flowerType} and ${diff.toFixed(2)} leva left.`)
    } else {
        console.log(`Not enough money, you need ${diff.toFixed(2)} leva more.`)
    }
}
